<launch>
<param name="/use_sim_time" value="true"/>
<param name="starting_x" type="double" value="0" />
<param name="starting_y" type="double" value="0" />
<param name="starting_th" type="double" value="0" />
<node pkg="first_project" type = "odom_node" name = "odom_node"/>
  
 <node pkg="tf" type="static_transform_publisher" name="sick_front_left" args="1.85 0.93 0.0 0.81 0.0 3.14 vehicle_centre sick_front_left 100" />
 <node pkg="tf" type="static_transform_publisher" name="sick_front_right" args="1.85 -0.93 0.0 -0.76 0.0 3.14 vehicle_centre sick_front_right 100" />
 <node pkg="tf" type="static_transform_publisher" name="sick_rear_left" args="-1.85 0.93 0.0 2.38 0.0 3.14 vehicle_centre sick_rear_left 100" />
 <node pkg="tf" type="static_transform_publisher" name="sick_rear_right" args="-1.75 -0.8 0.0 -2.30 0.0 3.14 vehicle_centre sick_rear_right 100" />

</launch>
